content/en/synthetics/guide/browser-tests-passkeys.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Using Passkeys (FIDO2) In Browser Tests
```

Comment:
@ruizb 
Get you please give a quick review for this doc page (file /browser-tests-passkeys.md)